contracts/cctp/SynapseCCTP.sol

```diff
@@ -159,7 +161,19 @@ contract SynapseCCTP is SynapseCCTPFees, Pausable, SynapseCCTPEvents, ISynapseCC
             burnToken,
             _destinationCaller(dstSynapseCCTP.bytes32ToAddress(), requestID)
         );
-        emit CircleRequestSent(chainId, nonce, burnToken, amount, requestVersion, formattedRequest, requestID);
+        // We want to emit the EOA address that initiated the transaction as "sender",
+        // so we use `tx.origin` instead of `msg.sender`.
+        // solhint-disable avoid-tx-origin
+        emit CircleRequestSent(
+            chainId,
+            tx.origin,
+            nonce,
+            burnToken,
+            amount,
+            requestVersion,
+            formattedRequest,
+            requestID
+        );
     }
 
     // TODO: guard this to be only callable by the validators?
```

Comment:
rm if non protected function?

Reply:
Resolved in [ec6ebb7](https://github.com/synapsecns/synapse-contracts/pull/242/commits/ec6ebb79ea5e5973dbfa232b11dfaed8b49e05e5)